src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/ObTableLSOperation.java

```diff
@@ -31,10 +32,18 @@ public class ObTableLSOperation extends AbstractPayload {
 
     private List<ObTableTabletOp> tabletOperations = new ArrayList<ObTableTabletOp>();
     private long                  lsId             = INVALID_LS_ID;                   // i64
+    private String tableName;
+
+    private long tableId = Constants.OB_INVALID_ID;;
+
+    // common column names for all single operation
+    private List<String> rowKeyNames = new ArrayList<>();
```

Comment:
Do we write the column name into this element?

Reply:
not used currently, will support soon (also add this comment in code).